docs/auth/multi-factor-auth.md

```diff
@@ -93,7 +101,7 @@ if (resolver.hints.length > 1) {
 }
 
 // Currently only phone based factors are supported
-if (resolver.hints[0].factorId === auth.PhoneMultiFactorGenerator.FACTOR_ID) {
+if (resolver.hints[0].factorId === authModule.PhoneMultiFactorGenerator.FACTOR_ID) {
```

Comment:
`PhoneMultiFactorGenerator` is supposed to be available as a direct / standalone import and should be used that way as documented in current firebase-js-sdk modular docs:

https://firebase.google.com/docs/auth/web/multi-factor

Reply:
AFAICT, the `PhoneMultiFactorGenerator` has not yet been properly exported for the modular API. I know we use it in the modular 2e2 tests, but my IDE goes on strike when it sees `import { PhoneMultiFactorGenerator } from "@react-native-firebase/auth"`. I'll update the docs anyways and then take a look at what's going on :)

Reply:
Looks like an export was missing from the `packages/auth/lib/modular/index.d.ts` file. I just opened #8508 to take care of it.

Reply:
can't believe I missed that 😆 - thanks for the PR, I'll see if I can get anything else queued for release and get it out shortly